src/current/cockroachcloud/configure-scim-provisioning.md

```diff
@@ -84,12 +96,14 @@ The following sections show how to manage the access of Okta users to your {{ si
 
 ### Assign a user or group
 
-To provision a user or group to {{ site.data.products.db }}, you assign the user or group to the app integration.
+To provision a user to {{ site.data.products.db }}, you assign the user or one of their groups to the app integration.
 
-{{site.data.alerts.callout_success}}
-After you assign a user to the app integration, changes that you make to the user's record in Okta, such as renaming the user or changing their email address, are automatically applied to the user's {{ site.data.products.db }} account.
+{{site.data.alerts.callout_info}}
+When you assign a group, accounts are provisioned for each of its members, but the group itself is not provisioned in {{ site.data.products.db }}. To provision groups to {{ site.data.products.db }} for role management, refer to [Automate Group Management](#automate-group-management).
```

Comment:
Again, maybe add Okta specificity here and that we don't know of behavior in other IdPs yet.